### DISCUSSION TITLE
[Performance] Trail with a list of 12+ items

### DISCUSSION THREAD

**Discussion by adilanchian** [pmndrs/react-spring#1344] [Support]
## 🤓 Question

Hey there 👋!

I noticed when using the `Trail` component from `renderprops`, that when trying to render more than 12 items in the a list, the animation actually never happens. Honestly not entirely sure where to start with this or if I am using Trail properly!

Any insight on this would be awesome :)

**Comment by tomdohnal** [comment]
Do you have a CodeSandbox with a repro?
Nonetheless, I was experiencing a similar issue https://github.com/pmndrs/react-spring/issues/1118.
